src/main/java/build/buildfarm/common/resources/BUILD

```diff
@@ -0,0 +1,54 @@
+load("@rules_proto//proto:defs.bzl", "proto_library")
+load("@rules_java//java:defs.bzl", "java_proto_library")
+
+package(default_visibility = ["//visibility:public"])
+
+proto_library(
+    name = "resource_proto",
+    srcs = ["resource.proto"],
+    deps = [
+        "@com_google_protobuf//:duration_proto",
+        "@com_google_protobuf//:timestamp_proto",
+        "@googleapis//:google_api_annotations_proto",
+        "@googleapis//:google_longrunning_operations_proto",
+        "@googleapis//:google_rpc_status_proto",
+        "@remote_apis//build/bazel/remote/execution/v2:remote_execution_proto",
+    ],
+)
+
+java_proto_library(
+    name = "resource_java_proto",
+    deps = [
+        ":resource_proto",
+    ],
+)
+
+java_library(
+    name = "ResourceParser",
```

Comment:
Rename this to `resources` - we have no capitalized names throughout, and our convention is to leverage bazel's basename target policy as much as possible.

Reply:
fixed